measures/ResStockArguments/resources/electrical_panel.rb

```diff
@@ -164,6 +222,22 @@ def convert_cooling_type(cooling_system_type, heat_pump_type)
     end
   end
 
+  def has_cooling(hvac_cooling_type)
+    if hvac_cooling_type != 'none'
+      return 1
+    else
+      return 0
+    end
+  end
+
+  def convert_heating_type(heat_pump_type)
```

Comment:
Mini-Split is not count here. In LBNL's major electric load calculation code, only ASHP is counted for 1 major electric load for heating and cooling. Since we have GHP in the latest ResStock, I further added GHP for 1 major electric load for heating and cooling.

Reply:
is_ducted_heat_pump_heating(heating_type)